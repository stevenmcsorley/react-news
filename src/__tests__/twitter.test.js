import React from 'react';
import { shallow } from 'enzyme';
import Twitter from '../components/news/TwitterFactory';

test("testing Twitter feed",  async () =>{

    const TwiiterComp = shallow(<Twitter />, 'div');
    expect(TwiiterComp).toMatchSnapshot();

})