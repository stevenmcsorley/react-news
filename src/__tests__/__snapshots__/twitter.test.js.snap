// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Twitter feed 1`] = `ShallowWrapper {}`;
